test/test.html

```diff
@@ -21,7 +21,7 @@
             console.error(error);
           };
           mocha.setup('bdd');
-          mocha.timeout(5000);
+          mocha.timeout(60000);
```

Comment:
Are we giving half of the timeout value from `mocha-headless-chrome` or should this be `120000` too?